scripts/prepdocs.py

```diff
@@ -205,10 +276,19 @@ async def main(strategy: Strategy, credential: AsyncTokenCredential, args: Any):
     parser.add_argument(
         "--tenantid", required=False, help="Optional. Use this to define the Azure directory where to authenticate)"
     )
+    parser.add_argument(
+        "--subscriptionid",
+        required=False,
+        help="Optional. Use this to define the Azure directory where to authenticate)",
+    )
     parser.add_argument(
         "--searchservice",
         help="Name of the Azure AI Search service where content should be indexed (must exist already)",
     )
+    parser.add_argument(
```

Comment:
Why don't we provision this if it doesn't exist? Is this only for external storage accounts?

Reply:
I am confused how this is used in the main.bicep vs. as args to this script

Reply:
It passed from main.bicep. Its needed by indexers to leverage system identity to authenticate